assets/src/blocks/Spreadsheet/SpreadsheetEditorScript.js

```diff
@@ -52,9 +56,9 @@ export class SpreadsheetEditor extends Component {
   renderEdit() {
     const {attributes, setAttributes} = this.props;
 
-    const toColorName = code => colors.find(color => color.color === code).name;
+    const toColorName = code => colors.find(color => color.color === code)?.name || 'grey';
 
-    const toColorCode = name => colors.find(color => color.name === name).color;
+    const toColorCode = name => colors.find(color => color.name === name)?.color || '#ececec';
```

Comment:
If the new identity styles are enabled, it should be a different color code

Reply:
Yeah I know, it's just that if for some reason we pass there a name that is not in the colors, the default color code will be `#ececec` which doesn't exist in the new identity color options... But that should never happen, I guess we can leave it as is